File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_Two/CPP/program3.cpp

#include <iostream>

int main () {

	int n, flag = 0;
	std::cout << ("Enter Km : ");
	std::cin >> n;

	if(n < 0)
		exit(0);
	std::cout << n << (" KM is equal to ") << n * 1000 << std::endl;
}

File: Daily Flash/week 4/day 1/18-DailyFlash_Solutions/3_Feb_Solutions_Two/Java/program3.java
import java.io.*;

class Program {
	public static void main(String[] args) throws IOException{
			
		int n = 0, flag = 0;
		System.out.printf("Enter Km : ");
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

		try{
			n = Integer.parseInt(br.readLine());
		}
		catch(NumberFormatException ne){
			System.exit(0);
		}

		if(n < 0)
			System.exit(0);
		System.out.printf("%d KM is equal to %dm\n", n, n * 1000);
	}
}
